ui/app/AppLayouts/Chat/views/MembersSelectorView.qml

```diff
@@ -97,7 +126,13 @@ MembersSelectorBase {
         }
 
         function processContact(publicKey) {
-            const contactDetails = Utils.getContactDetailsAsJson(publicKey, false)
+            let fullPubkey = publicKey
+            if (root.utilsStore.isCompressedPubKey(fullPubkey)) {
+                fullPubkey = root.utilsStore.getDecompressedPk(publicKey)
+            }
+            const contactEntry = d.getContactModelEntry(fullPubkey)
+            const contactDetails = contactEntry.contactDetails
+
             if (contactDetails.publicKey === "") {
```

Comment:
Based on the current ContactDetailsEntry implementation the public key is never empty unless `publicKey: d.sharedContactModelEntryLoader.publicKey` has an empty value. But this case is not what we want to validate here.

Looks to me like `ContactDetailsEntry` needs a `valid` or `available` flag. We can expose `ModelEntry.available` flag in the API.